scripts/check_cargo_deny

```diff
@@ -7,3 +7,7 @@ readonly SCRIPTS_DIR="$(dirname "$0")"
 source "$SCRIPTS_DIR/common"
 
 cargo deny check
+cargo deny --manifest-path=oak_abi/Cargo.toml check
```

Comment:
Maybe put something in `common` to declare a Bash array or two holding the master list of workspaces and/or standalone crates, and loop through that array in all of these scripts?  That way only one place should need changing for any future re-orgs.

Reply:
*cough* Rust runner *cough*